src/ex1/script.js

```diff
@@ -0,0 +1,98 @@
+const addButton = document.getElementById("addButton")
+const textInput = document.getElementById("input")
+const counterLists = document.getElementById("counterLists")
+const deleteAllLists = document.getElementById("deleteAllTasks")
+const errorMessage = document.getElementById("error")
+const allLists = document.getElementById("todolist")
+
+addButton.addEventListener("click", function () {
+  validationText()
+})
+
+textInput.addEventListener("keypress", function (e) {
+  if (e.key === "Enter") {
+    e.preventDefault()
+    validationText()
+  }
+})
+deleteAllLists.addEventListener("click", function () {
+  deleteAllTasks()
+})
+function addNewList() {
+  countLiAdded()
+  const input = document.getElementById("input")
+  const list = document.getElementById("todolist")
+  const li = document.createElement("li")
+  const deleteButton = document.createElement("button")
+  deleteButton.setAttribute("class", "dltbtn")
+  deleteButton.innerHTML = "X"
+  deleteButton.addEventListener("click", function () {
+    li.remove()
+    countLiDeleted()
+  })
+  li.addEventListener("mouseover", function () {
+    deleteButton.style.display = "block"
+    deleteButton.style.transition = "all 0.5s ease-in-out"
+  })
+  li.addEventListener("mouseout", function () {
+    deleteButton.style.display = "none"
+    deleteButton.style.transition = "all 0.5s ease-in-out"
+  })
```

Comment:
its a possible solution, but you can use here the CSS :hover selectors, its usually better to use them when possible, if you'r having problem with that contact me 😃